lib/arclight/custom_document.rb

```diff
@@ -37,5 +40,34 @@ def to_solr(solr_doc = {})
     def names
       [corpname, famname, name, persname].flatten.compact.uniq - repository
     end
+
+    # Combine subjets into one group from:
+    #  <controlaccess/><subject></subject>
+    #  <controlaccess/><function></function>
+    #  <controlaccess/><genreform></genreform>
+    #  <controlaccess/><occupation></occupation>
+    def access_subjects
+      subjects = search("//*[local-name()='subject' or local-name()='function' or local-name() = 'occupation' or local-name() = 'genreform']").to_a
```

Comment:
See https://github.com/sul-dlss/arclight/issues/11#issuecomment-295939352

Looks like we're good.